listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -98,6 +99,56 @@ def create_user_recording_recommendation_event(user_name):
     return jsonify(event_data)
 
 
+@user_timeline_event_api_bp.route('/user/<user_name>/timeline-event/create/notification', methods=['POST', 'OPTIONS'])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+def create_user_notification_event(user_name):
+    """ Post a message with a link on a user's timeline. Only approved users are allowed to perform this action.
+
+    The request should contain the following data:
+
+        {
+            "metadata": {
+                "message": <the message ot post, required>,
+                "link": <the link to include with the message, required>
+            }
+        }
+
+    :param user_name: The MusicBrainz ID of the user who is recommending the recording.
+    :type user_name: ``str``
+    :statuscode 200: Successful query, message has been posted!
+    :statuscode 400: Bad request, check ``response['error']`` for more details.
+    :statuscode 401: Unauthorized, you are not an approved user.
+    :statuscode 404: User not found
+    :resheader Content-Type: *application/json*
+
+    """
+    creator = validate_auth_header()
+    if creator["musicbrainz_id"] not in current_app.config['APPROVED_PLAYLIST_BOTS']:
+        raise APIUnauthorized("Only approved users are allowed to submit playlists made for someone else.")
```

Comment:
playlists?

Reply:
The intent of this endpoint was to allow troi-bot to post playlists but since we use notification everywhere hence the mistake. Fixed.